pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -957,8 +957,9 @@ public NamespaceService getNamespaceService() {
         return functionWorkerService;
     }
 
-    public WorkerService getWorkerService() {
-        return functionWorkerService.orElse(null);
+    public WorkerService getWorkerService() throws UnsupportedOperationException {
+        return functionWorkerService.orElseThrow(() -> new UnsupportedOperationException("Pulsar Functions worker " +
```

Comment:
what about:
"Pulsar Function Worker is not enabled, probably functionsWorkerEnabled is set to false"